code/BuddyTrackerBasic/src/main.cpp

```diff
@@ -72,11 +73,13 @@ void onReceive(uint8_t packetSize) {
 
         // read partial lat
         uint16_t lat_partial = 0;
+        // TODO make shift and cast does not cause problems. Safest to mask
```

Comment:
Always concerned when shifting bits and casting between sizes. Make sure that this always works as expected.